routes/index.js

```diff
@@ -3,11 +3,17 @@ const router = require('express').Router();
 const cfg = require('../config');
 const manifest = require('../dist/manifest.json');
 
-
 const assetUrl = (name) => {
   return `/static/${manifest[name]}`;
 };
 
+const User = require('../model/user');
+const userLocation = require('../model/user_location');
+const Image = require('../model/Image');
+
+User.hasOne(userLocation, { foreignKey: 'user_id' });
+User.hasOne(Image, { foreignKey: 'owner_id' });
```

Comment:
What are these functions be used for?
Is the routes a right place to handle with model functions?